main/unbound/APKBUILD

```diff
@@ -10,7 +10,8 @@ arch="all"
 license="BSD-3-Clause"
 depends="dns-root-hints dnssec-root"
 depends_dev="expat-dev"
-makedepends="$depends_dev libevent-dev openssl-dev python2-dev swig linux-headers"
+depends_migrate="/bin/sh apk-tools dns-root-hints openrc"
```

Comment:
Afaik this not an official APKBUILD var, hence I would prefix it with "_".

Reply:
I would not. This would complicate efforts to automatically handle `depends_*` variables when calculating depends in abuild.